.github/workflows/ci.yml

```diff
@@ -16,8 +18,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ "ubuntu-latest", "macos-latest"]
+        os: [ "ubuntu-latest"]
```

Comment:
Instead of removing macos-latest at all, I'd do ubuntu and macos but only for versions 3.7 and 3.10 for this pull-request CI